### ISSUE TITLE
SSO-Login Browser

### ISSUE THREAD

**Issue by lkstnr** [University-of-Potsdam-MM/login-provider#3]
Zur Zeit funktioniert der Login nur unter Cordova. Kriegen wir das auch im Browser hin?

**Comment by weihmann** [comment]
Stimmt, im Moment wird nur das `InAppBrowser`-Plugin verwendet. Sollte eigentlich auch im Browser funktionieren, schau ich mir an.
